### ISSUE TITLE
Add schema.org metadata for analysis packages

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#3019]
Which type should we use? https://schema.org/Code ?

**Comment by rreitmann** [comment]
DC.Type: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/dcmitype/Software/
Schema.org: https://schema.org/Code

**Comment by AndyDaniel1** [comment]
https://validator.schema.org/#url=https%3A%2F%2Fmetadata.fdz.dzhw.eu%2F

**Comment by tilovillwock** [comment]
@AndyDaniel1 I forgot to report back here. As far as I can tell both schemas are implemented for both data packages and analysis packages. The issue with the validator is that its backend might use a user agent string that is not honored in order to reroute its request to the pre-rendered version (SEO4Ajax) of the site instead of the dynamic version. That's the reason why we can see all those Angular placeholder variables.

I couldn't find any information regarding the schema.org user agent and was unable to find the access logs for the load balancer in AWS before the allocated time was up.

So in summary: the implementation is there. If there are issues regarding discoverability we might need to adjust the list of whitelisted user agents for the pre-rendered version.

**Comment by AndyDaniel1** [comment]
Memo: Gesis has offered support for the introduction of signposts to improve findability.
